scripts/copyFiles.js

```diff
@@ -154,13 +154,17 @@ async function run() {
     const packageData = await createPackageFile();
 
     const filesToCopy = [
-      // TODO: Improve if we want to use the core `copy-files.js` file
-      // use enhanced readme from workspace root for `@mui/material`
       './README.md',
+
+      // TODO: Remove `grid` folder to flatten the structure
```

Comment:
I'm waiting for the premium package to avoid conflicts, but I'm in favor of following the core structure with `mui-x-date-pickers`, `mui-x-data-grid` etc...